pwndbg/heap/__init__.py

```diff
@@ -28,8 +28,7 @@ def update():
     resolve_heap(is_first_run=True)
 
 
-@pwndbg.events.stop
-@pwndbg.events.new_objfile
+@pwndbg.events.exit
```

Comment:
Are we 100% sure about this change? When is .exit called?

Reply:
Yeah, I think it is, and I think I misuse the event before, the old code of this will be triggered frequently, but it seems that we only want to reset it when we want to debug a new process.